digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/GeometryTransform.scala

```diff
@@ -151,8 +151,8 @@ class VKMGeometryTransform {
 
   private def vkmBaseUrl = {
     val properties = new Properties()
-    properties.load(getClass.getResourceAsStream("/digiroad2.properties"))
-    properties.getProperty("digiroad2.VKMUrl") + "/viitekehysmuunnin/"
+    properties.load(getClass.getResourceAsStream("/env.properties"))
+    Digiroad2Properties.vkmUrl + "/viitekehysmuunnin/"
```

Comment:
vkmBaseUrl = Digiroad2Properties.vkmUrl + "/viitekehysmuunnin/"

Reply:
meinaatko näin?

  private def vkmBaseUrl = {
    val vkmBaseUrl = Digiroad2Properties.vkmUrl + "/viitekehysmuunnin/"
  }

Reply:
Tuskin tota tarvii olla funktion sisällä. Ihan String muuttuja joka koostaan noista osista. Yhdenmukaisuuden nimissä se vois silti olla noin vaikkapa  
```suggestion
    private def vkmBaseUrl = Digiroad2Properties.vkmUrl + "/viitekehysmuunnin/"
```